doubly_linked_list/doubly_linked_list.py

```diff
@@ -74,32 +75,39 @@ def add_to_tail(self, value):
     pass
 
   def remove_from_tail(self):
-    current_node = self.head
-
-    while current_node is not None:
-      current_node.prev.next = current_node.next
-      current_node.next.prev = current_node.prev
-
-      current_node = current_node.next
-    pass
+    if self.head == None and self.tail == None:
+      return None
+    temp = self.tail
+    if self.tail.prev is not None:
+      temp = self.tail
+      self.tail.delete()
```

Comment:
Great work so far Austin!, but there can be a more simplistic way of removing a pointer than rather deleting a whole node. Hint would be, rather than deleting a node, you would be rearranging the sequential ordering and removing the pointer of a current node.